### PR TITLE
transtermhp goes conda

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: tools/transtermhp/macros.xml [hunk 1]
```diff
@@ -3,8 +3,9 @@
   <xml name="requirements">
     <requirements>
       <requirement type="package" version="2.09">transtermhp</requirement>
-      <requirement type="package" version="1.65">biopython</requirement>
-      <requirement type="package" version="0.6.2">bcbiogff</requirement>
+      <requirement type="package" version="1.69">biopython</requirement>
+      <requirement type="package" version="0.6.4">bcbiogff</requirement>
+      <requirement type="package" version="2.7">python</requirement>
```
Comment (nsoranzo) [inline tools/transtermhp/macros.xml]: `transtermhp.py` is Python3 compatible, if that's why you added this last requirement.

File: tools/transtermhp/transtermhp.xml [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
+    <description>finds rho-independent transcription terminators in bacterial genomes</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <expand macro="stdio"/>
+    <command><![CDATA[
+        #if $reference_genome.source == 'history':
+            ln -s '$reference_genome.genome_fasta' genomeref.fa &&
+        #end if
```
Comment (nsoranzo) [inline tools/transtermhp/transtermhp.xml]: This absolutely comes down to personal preferences, but I like to start the content of `<command>` with no indentation (as it was before in this case), as we do with the `<help>` content, which is the other big block of CDATA text we have.
I think this makes it easier to properly indent the Cheetah directives. Worth discussing at https://github.com/galaxy-iuc/standards ?

Comment (peterjc) [inline tools/transtermhp/transtermhp.xml]: Good idea: https://github.com/galaxy-iuc/standards/issues/43

File: tools/transtermhp/macros.xml (hunk 1 above)
Comment (bgruening) [inline tools/transtermhp/macros.xml]: Unfortunately, it is not. `bytes` issues all over.

File: tools/transtermhp/transtermhp.xml (hunk 2 above)
Comment (bgruening) [inline tools/transtermhp/transtermhp.xml]: I would love to also intent the help section, but rst does not allow it :)

Comment (bgruening) [inline tools/transtermhp/transtermhp.xml]: I changes it :)

**Pull request merged**